B827EBFFFE67C526.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE67C526", 
+		"servers": [ 
+			{ 
+				"server_address": "asia.se.thethings.network", 
```

Comment:
This is not a valid router address, check the valid ones here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses